WordPress/gutenberg
# Update Grid layout design

## What?

Update grid layout design:

* Apply a border to the entire card rather than just the preview element. This emphasises the grid layout and better clarifies which fields are associated with each record.
* Hide field titles;
  * Consistent with List layout.
  * Decreases the overall card footprint to improve information density.
  * The title frequently adds noise as each field value is self explanatory.
  * Field titles can still be observed via tooltip when hovering / focusing the value.
* Apply gray background to preview placeholder
* Decrease space between cards.
* Templates page: Hide description field when empty. This avoids the awkward orphaned `–` which is only required in table layout.

### Pages

| Before | After |
| --- | --- |
| <img width="653" alt="Screenshot 2024-01-16 at 13 32 33" src="https://github.com/WordPress/gutenberg/assets/846565/a6563fd6-1be5-431f-8230-0158eb8304c4"> |  <img width="647" alt="Screenshot 2024-01-16 at 13 32 43" src="https://github.com/WordPress/gutenberg/assets/846565/9cbe99cf-1792-4fd8-bd8d-ff9f7b35ba90"> |
| <img width="657" alt="Screenshot 2024-01-16 at 13 34 07" src="https://github.com/WordPress/gutenberg/assets/846565/aacf872e-484e-4058-b264-4752ba607d45"> | <img width="662" alt="Screenshot 2024-01-16 at 13 34 16" src="https://github.com/WordPress/gutenberg/assets/846565/255f6797-0e1a-445a-922c-329e586f2018"> |

### Templates
| Before | After |
| --- | --- |
| <img width="650" alt="Screenshot 2024-01-16 at 13 45 27" src="https://github.com/WordPress/gutenberg/assets/846565/2e400359-b57a-42f1-9ff2-521346e1497e"> | <img width="657" alt="Screenshot 2024-01-16 at 13 44 44" src="https://github.com/WordPress/gutenberg/assets/846565/aca524ba-07ea-46df-a9d7-c3ae33ae1d4b"> |
| <img width="665" alt="Screenshot 2024-01-16 at 13 46 11" src="https://github.com/WordPress/gutenberg/assets/846565/8e3bb2bd-51fc-444d-a599-fc22f776e5e3"> | <img width="658" alt="Screenshot 2024-01-16 at 13 48 21" src="https://github.com/WordPress/gutenberg/assets/846565/7de11aee-f5e0-4c9b-879a-91983c1b3889"> |

### Patterns

| Before | After |
| --- | --- |
| <img width="652" alt="Screenshot 2024-01-16 at 13 49 42" src="https://github.com/WordPress/gutenberg/assets/846565/9c7661c9-8ddc-4835-8400-c25194a75f8a"> | <img width="657" alt="Screenshot 2024-01-16 at 13 50 02" src="https://github.com/WordPress/gutenberg/assets/846565/c6c587ff-93cf-43c3-bd91-14f78fe44110"> |
| <img width="650" alt="Screenshot 2024-01-16 at 13 50 37" src="https://github.com/WordPress/gutenberg/assets/846565/8e4d714c-bdf0-4c0f-a1ba-21e873917311"> | <img width="656" alt="Screenshot 2024-01-16 at 13 50 47" src="https://github.com/WordPress/gutenberg/assets/846565/91e4772d-a207-4e7e-bbbb-5bab84879d0b"> |


## Testing instructions
* Enable data views experiment.
* Browse Pages, Templates, and Pattern management views.
* Switch to grid layout and toggle field visibility.
* Ensure everything looks nice and works as expected.
